packages/lambda-xyz/src/wmts.capability.ts

```diff
@@ -1,9 +1,11 @@
-import { V, VNode, TileSetType } from '@basemaps/lambda-shared';
+import { V, VNode, TileSetType, VNodeElement } from '@basemaps/lambda-shared';
 import { EPSG, Projection } from '@basemaps/geo';
 import { ImageFormat } from '@basemaps/tiler';
 
 const { lat, lon } = Projection.Wgs84Bound;
 
+const MaxZoomLevel = 25;
```

Comment:
@palmerj  We have hear from several users that they do not want be restricted in terms of over-zooming. Often users are displaying much more accurate/detailed data over the top of our basemap, and dont want the contextual raster data to disappear just because its visual appearance is degraded. Is there another way to be transparent about the original resolution on the imagery?

Reply:
Hi @JBLINZ, the over-zooming is still possible in the GIS clients (QGIS, ArcGIS), it will just resample the last fetched zoom level of tiles. This change will just stop the fetching of new tiles. So users will be ok. I still think advertising these tile extra zoom levels is not recommended given the resolution. In the future should do this for vector tiles either, but again the client would over-zoom the vector if they want. Note: for web clients users will not be using WMTS endpoint, they will use the XYZ URL manually configuring the rest.